opensrp-chw-core/src/main/java/org/smartregister/chw/core/interactor/CoreCommunityRespondersInteractor.java

```diff
@@ -84,7 +85,12 @@ private void createAddCommunityResponderEvent(String jsonString) {
     }
 
     private void createRemoveCommunityResponderEvent(String baseEntityID) {
-        JSONObject form = FormUtils.getFormUtils().getFormJson(CoreConstants.JSON_FORM.COMMUNITY_RESPONDER_REGISTRATION_FORM);
+        JSONObject form = null;
+        try {
+            form = FormUtils.getFormUtils().getFormJson(CoreConstants.JSON_FORM.COMMUNITY_RESPONDER_REGISTRATION_FORM);
+        } catch (IOException e) {
+            Timber.e(e);
+        }
         AllSharedPreferences allSharedPreferences = Utils.context().allSharedPreferences();
         Event baseEvent = org.smartregister.chw.anc.util.JsonFormUtils.processJsonForm(allSharedPreferences, form.toString(), CoreConstants.TABLE_NAME.COMMUNITY_RESPONDERS);
```

Comment:
This would result on an NPE, we should execute all the code referencing the form within the try catch